library/core/src/char/convert.rs

```diff
@@ -23,14 +23,12 @@ use super::MAX;
 /// [`char`]s. `from_u32()` will return `None` if the input is not a valid value
 /// for a [`char`].
 ///
-/// [`char`]: ../../std/primitive.char.html
-/// [`u32`]: ../../std/primitive.u32.html
+/// [`char`]: crate::char
+/// [`u32`]: crate::u32
```

Comment:
You meant to link to the primitives, not the modules, I think.
```suggestion
```
(and also remove the trailing newline above, github won't let me add suggestions through deleted lines)